prepare/metrics/rag.py

```diff
@@ -416,3 +416,138 @@
     add_to_catalog(
         metric, f"metrics.rag.response_generation.{axis}.{base_metric}", overwrite=True
     )
+
+# end to end
+
+end_to_end_artifact_names = [
```

Comment:
suggest to remove this dict and use the keys of one of the other dicts

Reply:
(to avoid having to list the metric artifact names 3 times)

Reply:
fixed.